src/pybind/mgr/orchestrator.py

```diff
@@ -446,6 +456,16 @@ def __init__(self):
 
 
 class InventoryNode(object):
+    """
+    When fetching inventory, all Devices are groups inside of an
+    InventoryNode.
+    """
+
+    # MGR doesn't maintain Python module identity across MGR modules. Meaning
+    # isinstance() no longer works across MGR modules.
+    _uuid = '24dedb4a-f3d0-11e8-96e3-23ca4d957163'
+
```

Comment:
Maybe instead of uuids this should be some sort of enum defined in the orchestrator?

Reply:
yea, I'd like to remove these uuids again. This will make this PR much simpler.